modules/appeals_api/spec/support/shared_examples_contestable_issues.rb

```diff
@@ -2,183 +2,218 @@
 
 require AppealsApi::Engine.root.join('spec', 'spec_helper.rb')
 
-RSpec.shared_examples 'contestable issues index requests' do |options|
+# rubocop:disable Layout/LineLength
+RSpec.shared_examples 'Contestable Issues API v0 and Decision Reviews v1 & v2 shared request examples' do |base_path: ''|
+  # rubocop:enable Layout/LineLength
   describe '#index' do
-    context 'when using SSN header as veteran identifier' do
-      it 'GETs contestable_issues from Caseflow successfully' do
-        VCR.use_cassette("caseflow/#{options[:decision_review_type]}/contestable_issues") do
-          get_issues(ssn: '872958715', file_number: nil, options: options)
-          expect(response).to have_http_status(:ok)
-          json = JSON.parse(response.body)
-          expect(json['data']).not_to be nil
-        end
+    let(:ssn) { '872958715' }
+    let(:icn) do
+      # Contestable Issues API V0 requires ICN, others do not
```

Comment:
We may eventually want to split these examples out into smaller reusable chunks or refactor in some other way so that we don't need as much setup logic here, but I kept the same test cases as before in this file for now